packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -650,66 +652,154 @@ export class FluidDataStoreRuntime
 		);
 	}
 
-	public process(
-		message: ISequencedDocumentMessage,
+	/**
+	 * Process channel messages. The messages here are contiguous channel types messages in a batch for this data
+	 * store.
+	 * @param message - The core message with common properties for all the messages.
+	 * @param messageContents - The contents of the channel messages and their metadata.
+	 * @param local - true if the messages were originally generated by the client receiving it.
+	 */
+	private processChannelMessages(
+		message: ISequencedRuntimeMessageCore,
+		messageContents: IRuntimeMessageContents[],
 		local: boolean,
-		localOpMetadata: unknown,
 	) {
 		this.verifyNotClosed();
 
+		/*
+		 * Bunch contiguous messages for the same channel and send them together.
+		 * This is an optimization where DDSes can process a bunch of ops together. DDSes
+		 * like merge tree or shared tree can process ops more efficiently when they are bunched together.
+		 */
+		let previousAddress: string | undefined;
+		let previousMessageContents: IRuntimeMessageContents[] = [];
+
+		const sendPreviousBunch = () => {
```

Comment:
```suggestion
		const sendBunchedMessages = () => {
```
nit: Same as above (despite slight difference in how we're bunching here)